### DISCUSSION TITLE
ui library

### DISCUSSION THREAD

**Discussion by sadmann7** [sadmann7/tablecn#472] [Announcements]
i am planning to built an ui component repo in the following weeks.

there will be ci like shadcn that can be used to install all the components easily, also will add better docs.

also will add some more components i have been working with.

please let me know if anyone is interested in this

**Comment by Keith-Web3** [comment]
I could help out with this. Just let me know where I can come in

**Reply by sadmann7** [reply]
thanks, will let you know

**Comment by binlf** [comment]
Cool. would like to know when this starts...

**Comment by guibzo** [comment]
Nice

**Comment by laurent512** [comment]
big interest !!

**Comment by euotiniel** [comment]
Well, I'm interested in contributing. But what would be the differential?

**Comment by ashishxcode** [comment]
Sounds great, Loves to contribute by creating those components that are not available in shadcn UI will make more of a difference. 

Here is one example of a kind  https://shadcn-stepper.vercel.app by @damianricobelli

**Comment by Roman-Ka** [comment]
Hey @sadmann7 thanks for your great work, we are very interested in this! Can you share any updates if you managed to progress?

**Comment by sadmann7** [comment]
will start in the following month or two

**Reply by guibzo** [reply]
any news?

**Reply by sahandsn** [reply]
looking forward to it

**Comment by sadmann7** [comment]
just pushed an alpha for the [ui library](https://www.diceui.com/docs)

**Reply by guibzo** [reply]
Nice alpha!
Looking forward to the file drop component!
